delegate-admin/IC-Delegate-Admin.yml

```diff
@@ -0,0 +1,227 @@
+                      send(event, context, "FAILED", {'Error': str(e)})
+                      raise
+                  else:
+                      send(event, context, "SUCCESS", {})
+              elif request_type == 'Delete':
+                  send(event, context, "SUCCESS", {})
+              else:
+                  send(event, context, "SUCCESS", {})
+
+
+          def send(event, context, responseStatus, responseData, physicalResourceId=None, noEcho=False, reason=None):
+              responseUrl = event['ResponseURL']
+
+              print(responseUrl)
+
+              responseBody = {
+                  'Status': responseStatus,
+                  'Reason': reason or "See the details in CloudWatch Log Stream: {}".format(context.log_stream_name),
+                  'PhysicalResourceId': physicalResourceId or context.log_stream_name,
+                  'StackId': event['StackId'],
+                  'RequestId': event['RequestId'],
+                  'LogicalResourceId': event['LogicalResourceId'],
+                  'NoEcho': noEcho,
+                  'Data': responseData
+              }
+
+              json_responseBody = json.dumps(responseBody)
+
+              print("Response body:")
+              print(json_responseBody)
+
+              headers = {
+                  'content-type': '',
+                  'content-length': str(len(json_responseBody))
+              }
+
+              try:
+                  response = http.request(
+                      'PUT', responseUrl, headers=headers, body=json_responseBody)
+                  print("Status code:", response.status)
+
+              except Exception as e:
+
+                  print("send(..) failed executing http.request(..):", e)
+      Role:
+        Fn::GetAtt:
+          - lambdaExecutionRole957698A4
+          - Arn
+      FunctionName: delegate-IC-Admin
+      Handler: index.handler
+      MemorySize: 1024
+      Runtime: python3.8
+      Timeout: 30
+  lambdaInvokeF19AF3C4:
+    Type: Custom::AWS
+    Properties:
+      ServiceToken:
+        Fn::GetAtt:
+          - delegateAdminFunctionC209BD88
+          - Arn
+      delegate: !Sub ${delegate}
+      account_id: !Sub ${accountid}
+      physicalResourceId: JobSenderTriggerPhysicalId
+      InstallLatestAwsSdk: true
+    DependsOn:
+      - lambdaInvokeCustomResourcePolicyC17933EB
+    UpdateReplacePolicy: Delete
+    DeletionPolicy: Delete
+  lambdaInvokeCustomResourcePolicyC17933EB:
+    Type: AWS::IAM::Policy
+    Properties:
+      PolicyDocument:
+        Statement:
+          - Action: lambda:InvokeFunction
+            Effect: Allow
+            Resource:
+              Fn::GetAtt:
+                - delegateAdminFunctionC209BD88
+                - Arn
+        Version: "2012-10-17"
+      PolicyName: lambdaInvokeCustomResourcePolicyC17933EB
+      Roles:
+        - Ref: AWS679f53fac002430cb0da5b7982bd2287ServiceRoleC1EA0FF2
+  AWS679f53fac002430cb0da5b7982bd2287ServiceRoleC1EA0FF2:
```

Comment:
I would recommend to use human-readable resource name

Reply:
I used CDK to generate that template. I'll change the names. Thanks